assets/src/edit-story/components/inspector/colorPickerProvider/colorPickerProvider.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { useState, useCallback, useRef } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ColorPicker from '../../colorPicker';
+import Context from './context';
+
+const Outer = styled.div`
+  height: 100%;
```

Comment:
`width: 100%` as well? I also made a note above - maybe it'd be easier to manage this if we just portal-ed the popup onto body or another higher-level root?